File: commons/src/main/java/fr/deroffal/bibliotheque/commons/mapping/MapperConfiguration.java
package fr.deroffal.bibliotheque.commons.mapping;

import static org.mapstruct.ReportingPolicy.WARN;

import org.mapstruct.MapperConfig;

//TODO Erreur de compilation sur le passage en java 17 si unmappedTargetPolicy = ERROR
@MapperConfig(componentModel = "spring", unmappedTargetPolicy = WARN)
public interface MapperConfiguration {
}

File: livre/src/main/kotlin/fr/deroffal/bibliotheque/livre/configuration/MapperConfiguration.kt
package fr.deroffal.bibliotheque.livre.configuration

import org.mapstruct.MapperConfig
import org.mapstruct.ReportingPolicy

//TODO Erreur de compilation sur le passage en java 17 si unmappedTargetPolicy = ERROR
@MapperConfig(componentModel = "spring", unmappedTargetPolicy = ReportingPolicy.WARN)
interface MapperConfiguration {
}
